cmd/rule-evaluator/main.go

```diff
@@ -542,8 +584,12 @@ func sendAlerts(s *notifier.Manager, projectID string, externalURL *url.URL) rul
 			}
 			if externalURL != nil {
 				if externalURL.String() == googleCloudBaseURL.String() {
-					// If it's a GCM link (default), create the full URL for the alert.
-					a.GeneratorURL = googleCloudLink(projectID, expr, alert.FiredAt, alert.FiredAt.Add(-time.Hour)).String()
+					if projectID == "" {
```

Comment:
Should we add a commentary on why project ID can be empty (it's for noop state when no rules are configured).